src/main/java/core/basesyntax/main/Main.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.main;
+
+import core.basesyntax.dataparser.DataParser;
+import core.basesyntax.dataparser.DataParserStrategyImpl;
+import core.basesyntax.dataparser.ParserModels;
+import core.basesyntax.dataprocessor.DataProcessor;
+import core.basesyntax.dataprocessor.DataProcessorStrategyImpl;
+import core.basesyntax.dataprocessor.ProcessSchema;
+import core.basesyntax.dstdao.DestinationDao;
+import core.basesyntax.dstdao.DestinationDaoStrategyImpl;
+import core.basesyntax.dstdao.DestinationTypes;
+import core.basesyntax.reportgenerator.ReportGenerator;
+import core.basesyntax.reportgenerator.ReportGeneratorStrategyImpl;
+import core.basesyntax.reportgenerator.ReportTemplate;
+import core.basesyntax.srcdao.SrcDao;
+import core.basesyntax.srcdao.SrcDaoStrategyImpl;
+import core.basesyntax.srcdao.SrcTypes;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
```

Comment:
Remove unnecessary strategies and refactor this class accordingly. Your task is to implement a single strategy based on operation types, but such a strategy is not present in your solution.